### ISSUE TITLE
训练高度为64的图片会有问题

### ISSUE THREAD

**Issue by PKQ1688** [Sierkinhane/crnn_chinese_characters_rec#232]

**Comment by Sierkinhane** [comment]
具体的错误是什么？

**Comment by PKQ1688** [comment]
cnn 出来得到的特征 h 不为 1 ，传入LSTM会有问题。

**Comment by Sierkinhane** [comment]
可以展开成一个高度为1的特征序列

**Comment by PKQ1688** [comment]
'''
conv1 = conv[:, :, 0, :]
conv2 = conv[:, :, 1, :]
conv = torch.stack((conv1, conv2), 1)
conv = torch.reshape(conv, (b, h * c, w))
''' 我直接reshape的结果不能收敛

**Comment by Sierkinhane** [comment]
conv = conv.flatten(start_dim=1,end_dim=2).unsqueeze(dim=2)

**Comment by Sierkinhane** [comment]
按道理这样做是不对的，特征没有和字符对应起来

**Comment by PKQ1688** [comment]
是的 所以是不是有可能造成不收敛

**Comment by Sierkinhane** [comment]
建议你 增加一次卷积

**Comment by PKQ1688** [comment]
ks = [3, 3, 3, 3, 3, 3, 2, 1]
        ps = [1, 1, 1, 1, 1, 1, 0, 0]
        ss = [1, 1, 1, 1, 1, 1, 1, 1]
        nm = [64, 128, 256, 256, 512, 512, 512, 512, 512, 1024]

**Comment by PKQ1688** [comment]
我在后面新增了一个convRelu 参数如上 可以吗

**Comment by Sierkinhane** [comment]
保证最后的特征输出高度为1就行

**Comment by PKQ1688** [comment]
我之前增加了pooling层，也无法收敛

**Comment by eeewhe** [comment]
再加一层卷积池化就行吧
